### ISSUE TITLE
Interaction failed on remove command

### ISSUE THREAD

**Issue by X-Legendary** [wotanut/status#4]
When using the `/remove` command, without supplying the `user`, the bot doesn't reply. My guess is that just using `/remove` would remove all tracked bots in the current guild it is used in.

**Comment by wotanut** [comment]
I think the issue might be that it's trying to find a guild ID in the database that does not exist, hopefully a try catch around
```python
if user == None:
    collection.update_many( { }, { '$unset': { str(ctx.guild.id): '' } } )
    await ctx.respond(f"Removed all mentions of {ctx.guild.name} from the database")
else:
    collection.update_one({"_id": user.id}, {"$unset" : {f"{ctx.guild.id}":""}})
    await ctx.respond(f"Removed {user.mention} from the database")
```
    
Should fix it

Either that or there's something wrong with the collection.update_one method

**Comment by X-Legendary** [comment]
I see, when you push the code change into production, I shall test again (the issue should be **hopefully** fixed), and see what happens. I'll also open a PR if I find a way to fix it.

**Comment by wotanut** [comment]
Fixed with commit 6b1bc210e0e44dfccf10fc554c7c9dc89c2f1640 (https://github.com/wotanut/status/commit/6b1bc210e0e44dfccf10fc554c7c9dc89c2f1640)
